### PR TITLE
Pre app launch patches hack2019

### PR DESCRIPTION
**Pull request by kbuci**

### CODE REVIEW

File: app/server/services/sendgrid_email.js [hunk 1]
```diff
@@ -54,37 +54,42 @@ function sendOne(template, email, templateData, callback) {
...
  * @param  {[type]}     token    [description]
  * @param  {Function} callback [description]
  * @return {[type]}            [description]
  */
-emailService.sendVerificationEmail = function (email, token, callback) {
+emailService.sendVerificationEmail = function (email, firstName, token, callback) {
   var verifyUrlData = {
-    url: `${ROOT_URL}/verify/${token}`
+    url: `${ROOT_URL}/verify/${token}`,
+    firstName
```
Comment (Jeremyyang920) [inline app/server/services/sendgrid_email.js:65]: shouldnt this be firstName:firstname.
It needs to know what it's replacing.

File: app/server/services/sendgrid_email.js [hunk 2]
```diff
@@ -54,37 +54,42 @@ function sendOne(template, email, templateData, callback) {
...
+ * @param  {string}   firstName
  * @param  {[type]}   token    [description]
  * @param  {Function} callback [description]
  */
-emailService.sendPasswordResetEmail = function (email, token, callback) {
+emailService.sendPasswordResetEmail = function (email, firstName, token, callback) {
   var resetUrlData = {
-    url: `${ROOT_URL}/reset/${token}`
+    url: `${ROOT_URL}/reset/${token}`,
+    firstName
```
Comment (Jeremyyang920) [inline app/server/services/sendgrid_email.js:80]: same as above

File: app/server/services/sendgrid_email.js [hunk 3]
```diff
@@ -54,37 +54,42 @@ function sendOne(template, email, templateData, callback) {
...
 /**
  * Send a password recovery email.
  * @param  {string|string[]}  email         [Email address(s) to send to]
+ * @param  {string}   firstName
  * @param  {Function} callback [description]
  */
-emailService.sendPasswordChangedEmail = function (email, callback) {
-  sendOne(PASSWORD_CHANGED_EMAIL_TEMPLATE, email, {}, callback);
+emailService.sendPasswordChangedEmail = function (email, firstName, callback) {
+  sendOne(PASSWORD_CHANGED_EMAIL_TEMPLATE, email, { firstName }, callback);
```
Comment (Jeremyyang920) [inline app/server/services/sendgrid_email.js]: same as above

**Review by Jeremyyang920 — CHANGES_REQUESTED** [review]

File: app/server/services/sendgrid_email.js (hunk 1 above)
Comment (kbuci) [reply app/server/services/sendgrid_email.js:65]: I remember Chris pointed out that ES6 allows for this shortcut:
Look for "Object Literals and Concise parameters" section in https://medium.freecodecamp.org/make-your-code-cleaner-shorter-and-easier-to-read-es6-tips-and-tricks-afd4ce25977c

**Review by Jeremyyang920 — APPROVED** [review]

**Pull request merged**